intg/src/main/java/org/apache/atlas/model/instance/AtlasEntity.java

```diff
@@ -74,6 +74,7 @@ public class AtlasEntity extends AtlasStruct implements Serializable {
 
     /**
      * Status of the entity - can be active or deleted. Deleted entities are not removed from Atlas store.
+     * Purged status also used as Relationship status when entity is Hard Deleted/Purged
```

Comment:
Did not get this?

Reply:
AtlasEntity status is used as Relationship status as well, this is used to signify that this is a Hard Delete/Purge event in Kafka for the consumer.